# Global Temperatures

In [1]:
import pandas as pd

In [2]:
data = pd.read_csv("./data/data.csv")
data.head()

Year  Temp
0  1880 -0.19
1  1881 -0.10
2  1882 -0.08
3  1883 -0.19
4  1884 -0.26

The file data.csv contains the average annual global temperatures from 1880 to 2015.
The temperatures are expressed as the difference between the annual temperature in °C and the average temperature between 1950 and 1980. This is referred to as temperature anomaly.
The file contains two variables: Year and Temp. The variable Temp represents the temperature anomaly.

In [3]:
temperature_1901 = data.Temp[data.Year == 1901]
temperature_2000 = data.Temp[data.Year == 2000]
temperature_rise = temperature_2000.reset_index(drop=True) - temperature_1901.reset_index(drop=True)
temperature_rise

0    0.56
Name: Temp, dtype: float64

In [4]:
sort_index = data.Temp.sort_values(ascending=False).index
top15_years = data.Year[sort_index[:15]]
top15_years

135    2015
134    2014
130    2010
125    2005
127    2007
133    2013
129    2009
118    1998
126    2006
122    2002
132    2012
123    2003
131    2011
124    2004
121    2001
Name: Year, dtype: int64

In [5]:
h21 = (top15_years >= 2001).sum()
h21

14